tests/unit/ReportUtilsTest.ts

```diff
@@ -270,6 +270,30 @@ describe('ReportUtils', () => {
                 });
             });
         });
+
+        describe('ParentReportAction is', () => {
```

Comment:
@roryabraham here's an example `getReportName` test I am planning to add more of, for the other examples in this PR - I'll probably even add more tests but can you at least let me know if this is what you're thinking for tests / any feedback on this one?

Reply:
I was thinking more along the lines of UI tests rendering `<ReportActionItem>` using `@testing-library/react-native`. There are only limited examples in the repo today

Reply:
Hmm in that case since there's not much to go on right now, i'd prefer taking this as a follow-up for a contributor or C+ to write up b/c it's going to take me a bit to figure out how that works & that'll keep delaying this PR from getting merged

Reply:
> it's going to take me a bit to figure out how that works & that'll keep delaying this PR from getting merged

Up to you. I'd encourage you to take the time to learn and write the tests, because I don't see this PR as so urgent that we can't wait for tests. Just my opinion though (informed by my observed experience that 90% of the time when we "follow up to add tests", that part just never happens) 🙂

Reply:
I feel confident that I can get unit tests written externally, so I created this follow-up https://github.com/Expensify/App/issues/50351

I believe this will be more time efficient because I have more important issues to focus on this week